ingest-app/src/cmr/ingest/data/bulk_update.clj

```diff
@@ -38,9 +46,10 @@
   (get-bulk-update-task-status
     [db task-id]
     ;; Returns a status for the particular task
-    (su/find-one db (su/select [:status :status-message]
-                     (su/from "bulk_update_task_status")
-                     (su/where `(= :task-id ~task-id)))))
+    (let [status (su/find-one db (su/select [:status :status-message]
+                                   (su/from "bulk_update_task_status")
```

Comment:
The lines `(su/from ...` and `(su/where ...` should be indented to align with the first arg, `[:status ...`.